app/views/case_contacts/_form.html.erb

```diff
@@ -214,13 +214,26 @@ readonly: true %>
     <% end %>
   </div>
 
-  <div class="card-style-1 pl-25 mb-10">
-    <h2 class="mb-3"><%= form.label :notes, "5. Enter Notes" %></h2>
-    <div class="cc-italic mb-3">
-      Please refer to individuals by their roles instead of by their names. Ex: My supervisor joined me for a call with the social worker to discuss my youth.
+  <div class="card-style-1 pl-25 pr-25 mb-10">
+    <div class="d-flex justify-content-between mb-3" data-controller="icon-toggle" data-icon-toggle-icons-value='["lni-chevron-up", "lni-chevron-down"]'>
+      <h2 class=""><%= form.label :notes, "5. Additional notes (optional)" %></h2>
```

Comment:
I know the ticket asks for the number to be 7 here, but I think 4 followed by a 7 is misleading in the moment. This can be fixed when more work is done on the other related issues.
<img width="809" alt="Screenshot 2023-12-23 at 10 44 50 AM" src="https://github.com/rubyforgood/casa/assets/50247514/3bf904c6-62c9-4f9f-8657-da0452bd8440">